ui/components/editor-channel-dialog.vue

```diff
@@ -472,10 +472,10 @@ export default {
         Pan: /^(?:Pan|Horizontal Movement)$/i,
         Tilt: /^(?:Tilt|Vertical Movement)$/i,
         PanTiltSpeed: /^(?:Pan ?\/? ?Tilt|Movement) (?:Speed|Time|Duration)$/i,
-        WheelSlot: /Wheel$|Disk$|Gobos? ?\d*$/i,
         WheelShake: /\bShake\b/i,
         WheelSlotRotation: /Gobo ?\d* (?:Rotation|Index)/i,
         WheelRotation: /Wheels? (?:Rotation|Index)/i,
```

Comment:
```suggestion
         WheelRotation: /Wheels? ?\d* (?:Rotation|Index)/i,
```

Otherwise, `Color Wheel 1 Rotation` is misinterpreted as `WheelSlot` (whereas `Color Wheel Rotation` is still interpreted as `WheelRotation`).